getting_started.rst

```diff
@@ -63,7 +63,8 @@ If you don't use conda (see above), create a virtual environment::
     Depending on your platform, the Python 3 interpreter could be invoked by
     ``python`` instead. This is the case for conda on Windows for example.
 
-Install ``aiohttp`` in the virtual environment::
+Activate the virtual environment and install ``aiohttp`` inside:
 
-    ./venv/bin/python -m pip install -U aiohttp
+    source ./venv/bin/activate
+    pip install -U aiohttp
```

Comment:
We should choose one approach and use it everywhere in the documentation. I see three options:

```
$ venv/bin/python examples/hello_world.py
```

```
$ alias python=$(pwd)/venv/bin/python
$ python examples/hello_world.py
```

```
$ source venv/bin/activate
$ python examples/hello_world.py
```

What's do you prefer ? 
I think sourcing the virtualenv is the cleanest approach.